views/includes/page-templates/managing-officer-corporate.html

```diff
@@ -78,15 +78,10 @@ <h2 class="govuk-heading-l">Who can we contact about this managing officer?</h2>
       {% include "includes/inset-text.html" %}
 
       {% include "includes/save-and-continue-button.html" %}
-
-      {% if pageParams.isRegistration %}
-        {% set removePathURL = OE_CONFIGS.MANAGING_OFFICER_CORPORATE_URL + OE_CONFIGS.REMOVE + "/" + id %}
-      {% else %}
-        {% set removePathURL = OE_CONFIGS.UPDATE_MANAGING_OFFICER_CORPORATE_URL + OE_CONFIGS.REMOVE + "/" + id %}
-      {% endif %}
-      {% set nameToBeRemoved = name %}
-      
+ 
       {% if not entity_number %} 
+        {% set removePathURL = OE_CONFIGS.MANAGING_OFFICER_CORPORATE_URL + OE_CONFIGS.REMOVE + "/" + id %}    
+        {% set nameToBeRemoved = name %}
```

Comment:
Should there be an else for the update case?

Reply:
No else required for the update case. Remove is done through another approach for update.

Only display remove button for registration and not for update